chromadb/telemetry/opentelemetry/__init__.py

```diff
@@ -61,6 +62,13 @@ def __init__(self, system: System):
 granularity: OpenTelemetryGranularity = OpenTelemetryGranularity("none")
 
 
+def _transform_attributes(
```

Comment:
Since we're not calling this, let's delete it.

Reply:
Done